src/main/java/com/faforever/client/api/FafApiAccessorImpl.java

```diff
@@ -257,6 +257,7 @@ public void uploadMod(Path file, ByteCountListener listener) {
   @Override
   public void uploadMap(Path file, boolean isRanked, ByteCountListener listener) {
     MultiValueMap<String, Object> multipartContent = createFileMultipart(file, listener);
+    multipartContent.add("metadata", "{\"isRanked\":\"" + isRanked + "\"}");
```

Comment:
I'm a bit confused. That doesn't look like a solution to any of the problems I have encountered so far. Can you give specific info which error message you intend to fix here?